cmd/vcfToFa/vcfToFa.go

```diff
@@ -117,11 +163,13 @@ func usage() {
 
 func main() {
 	var expectedNumArgs int = 3
-	var useAlt *bool = flag.Bool("useAlt", false, "Retains only biallelic variants in the output file.")
+	var useAlt *bool = flag.Bool("useAlt", false, "Creates a new sequence using the alternative base at every site (only uses biallelic SNPs).")
+	var useSamples *bool = flag.Bool("useSamples", false, "Creates new sequences using the genotype of that individual (only uses biallic SNPs).")
 	var multiFaMode *bool = flag.Bool("multiFaMode", false, "Use a multiFa reference chromosome. Variants will be placed in reference"+
 		"positions with respect to gaps. Input vcf must be sorted when using multiFaMode. Appends a new sequence with the updated sequence from the reference sequence,"+
 		"defined as the first sequence in the multiFa. Currently only supports useAlt.")
 	var multiFaChromName *string = flag.String("multiFaChromName", "", "Identifies the chrom of the multiFa file. Only variants matching this chrom name will be used.")
+	var haploid *bool = flag.Bool("haploid", false, "Only make one sequence per individual in the vcf, instead of two.  The first genotype will be used.")
```

Comment:
Is it worth specifying that only diploid and haploid are accepted? If I remember correctly, at some point Craig was helping someone use gonomics to parse vcfs of organisms with other ploidies.
And what is "the first genotype"? I thought if a sample is haploid there would only be 1 entry in its genotype column?

Reply:
In the current PR, yes, we only handle haploid or diploid samples. For other polyploid samples, we'll need an additional argument that parses samples in a for loop for an arbitrary number of alleles. This is sort of a quicker fix for some specific short term use cases

Reply:
I have added a note to the usage statement that we are assuming diploid, unless the haploid flag is given.